Clustering/Hdbscan/src/test/java/org/tribuo/clustering/hdbscan/TestHdbscan.java

```diff
@@ -0,0 +1,228 @@
,4,4,4,3,3,5,5,3,4,4,5,5,3,5,3,5,4,5,4,3,3,5,3,5,3,3,3,4,3,4,5,5,5,4,4,4,5,5,4,3,4,5,4,5,3,5,0,5,4,5,4,5,5,5,4,4,5,3,4,3,5,5,5,3,5,5,3,5,3,5,4,4,4,5,5,5,4,3,5,5,3,3,5,3,5,4,3,4,5,5,5,5,5,5,3,4,4,4,5,5,5,3,5,5,5,3,3,3,5,3,5,5,4,4,4,4,5,5,5,5,5,4,3,5,3,5,0,3,5,5,5,3,5,5,4,0,4,0,3,5,5,3,3,5,5,3,3,5,5,4,3,3,5,5,5,5,3,5,5,5,4,5,4,5,5,3,3,4,5,3,4,5,5,5,5,5,5,5,5,5,3,5,4,5,4,5,0,5,5,4,5,4,4,3,5,3,4,3,3,5,3,3,5,4,5,5,5,5,4,0,3,3,4,4,3,5,4,3,3,4,5,5,4,3,4,5,4,5,5,5,4,5,3,3,3,4,4,4,3,4,5,5,5,3,5,3,5,5,4,3,3,5,4,5,3,3,5,4,3,3,5,4,5,5,4,5,4,5,3,3,5,5,5,0,5,3,4,3,5,4,5,5,5,5,5,5,5,4,4,5,3,0,5,4,4,4,5,5,5,3,5,5,5,4,5,4,5,5,5,5,5,0,5,5,3,3,5,3,4,5,3,4,4,5,3,5,5,3,5,3,5,5,5,5,3,4,3,3,5,3,4,4,3,5,4,5,5,4,4,3,4,5,5,4,5,3,5,3,4,3,5,0,4,4,4,5,4,5,4,3,5,3,0,4,4,5,5,5,3,4,3,5,5,3,3,3,4,5,5,5,3,4,5,0,4,5,5,3,5,3,0,3,5,4,5,5,4,5,5,5,5,5,4,4,4,3,4,5,4,5,4,5,4,4,4,3,4,5,4,5,3,5,5,3,4,5,5,5,5,4,5,3,3,3,3,5,4,5,5,5,3,5,3,5,5,5,3,4,5,5,5,5,5,4,5,5,5,3,0,5,3,3,5,5,3,5,5,4,3,3,3,3,0,3,4,5,3,4,5,5,4,5,3,3,3,5,5,3,4,3,5,4,5,5,5,3,5,3,5,5,5,5,5,5,4,3,5,3,3,4,5,5,4,4,3,5,4,5,4,5,5,5,5,5,3,3,0,5,3,5,4,5,5,5,5,3,5,5,3,5,4,4,5,3,5,4,3,4,5,3,5,5,4,3,5,3,3,5,5,5,3,3,3,5,5,5,3,4,5,5,5,5,5,5,5,4,5,3,5,4,3,3,3,5,5,5,5,4,5,4,5,4,5,4,3,5,0,5,5,0,4,5,5,4,3,5,3,4,5,3,5,5,5,3,5,5,5,4,5,5,3,3,5,5,5,5,4,5,4,5,3,4,5,4,5,5,0,5,4,5,3,3,4,5,3,5,5,5,3,4,3,4,3,4,4,5,3,5,5,0,3,4,3,5,5,4,5,5,4,5,3,5,3,5,5,3,5,3,4,5,5,3,5,4,5,5,5,4,4,5,5,5,3,4,3,5,4,4,5,5,5,3,3,5,4,5,5,3,3,5,4,5,4,4,4,3,3,3,5,3,3,3,5,3,5,5,5,4,5,5,3,4,3,3,3,5,3,5,4,4,5,3,0,5,3,3,5,5,4,5,3,4,4,5,5,4,4,5,3,4,4,5,5,5,5,3,5,3,3,5,5,3,5,5,3,3,3,5,5,4,3,5,4,4,3,5,5,3,0,4,4,4,3,4,5,5,5,5,3,5,3,5,3,5,4,5,3,4,3,5,5,5,5,4,3,5,5,3,5,5,4,4,5,3,5,3,5,4,3,5,5,5,5,4,4,3,5,4,5,5,5,5,5,5,3,5,4,5,5,4,5,3,4,4,5,3,4,5,5,5,3,4,4,4,5,5,4,5,3,3,4,5,3,5,5,4,5,5,5,4,5,5,3,4,5,5,0,4,4,3,5,5,5,4,4,5,5,4,5,5,4,3,4,3,5,0,3,5,4,5,3,3,5,4,3,5,5,5,4,5,3,5,5,5,4,5,5,0,5,5,5,4,3,4,5,0,5,5,3,5,0,5,5,5,5,5,5,4,3,5,3,4,0,5,5,0,5,5,3,5,5,3,5,5,5,5,4,3,3,5,4,3,3,3,5,5,5,5,4,3,3,5,5,5,3,5,3,3,4,5,5,5,4,5,5,5,5,5,5,5,5,4,5,3,5,5,4,5,5,4,3,5,5,5,5,5,4,5,0,5,3,5,3,3,5,5,4,5,5,5,5,4,5,4,5,5,4,5,5,0,4,5,5,4,4,5,5,5,5,4,5,5,5,3,5,5,5,4,5,4,4,5,4,4,4,5,4,3,3,3,4,3,5,4,3,5,5,3,3,5,3,4,3,5,5,5,5,3,5,5,3,5,4,3,0,5,4,4,3,5,4,4,3,5,4,5,5,5,4,3,4,5,5,4,3,3,5,5,5,5,5,5,3,5,3,5,5,4,4,5,5,3,5,5,5,5,5,5,3,5,3,0,5,3,3,5,5,5,5,3,3,4,3,4,5,5,3,5,4,5,4,3,3,4,5,5,3,5,4,3,5,3,3,4,0,5,5,5,4,4,5,3,3,5,5,5,5,5,5,3,5,5,5,4,4,5,5,5,3,5,5,3,0,4,4,3,4,4,4,5,0,5,3,5,5,4,5,5,3,5,4,3,5,5,5,4,4,5,5,5,5,3,5,4,4,5,4,5,5,5,3,4,3,5,5,4,5,4,4,0,4,4,4,4,4,3,5,3,5,4,5,4,4,3,3,3,5,5,5,5,5,3,5,3,5,4,5,4,4,4,3,3,3,3,5,5,5,4,3,5,3,5,3,4,5,5,5,4,5,3,3,3,4,3,3,5,4,5,3,5,3,5,5,3,5,3,5,3,4,4,4,3,5,5,5,3,4,4,3,5,3,4,4,4,4,5,3,5,5,4,5,5,5,5,3,5,5,5,4,3,4,4,5,4,4,3,5,5,5,5,3,5,5,5,4,4,3,3,3,5,4,3,0,4,4,5,0,5,5,3,3,4,5,4,5,3,3,5,4,5,5,3,5,4,5,5,5,5,4,4,5,5,5,5,5,0,3,4,5,4,4,5,3,4,5,5,5,3,3,3,0,0,3,3,5,3,3,5,5,3,5,5,5,4,5,4,3,4,3,0,3,5,4,4,3,5,4,4,4,3,5,5,3,4,4,3,3,4,5,4,3,3,3,4,5,3,3,4,4,5,3,5,5,3,5,4,4,5,3,3,3,3,4,0,5,4,5,5,4};
```

Comment:
Is it worth putting these cluster labels into the csv files and loading them that way?

Reply:
I did consider this, but thought it might be cumbersome to have a few different csv files to refer to. I preferred having the values right in the test, but am not strongly opposed to using csv files either.

Reply:
Ok, that's fine. I agree it makes it easier to use the same test data for different algorithms.